install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -793,4 +793,6 @@ VALUES (NULL, 'actionAdminAdminPreferencesControllerPostProcessBefore', 'On post
        (NULL, 'actionAddressGridPresenterModifier', 'Modify address grid template data',
         'This hook allows to modify data which is about to be used in template for address grid', '1'),
        (NULL, 'actionCreditSlipGridPresenterModifier', 'Modify credit slip grid template data',
-        'This hook allows to modify data which is about to be used in template for credit slip grid', '1');
+        'This hook allows to modify data which is about to be used in template for credit slip grid', '1'),
+       (NULL, 'displayAdditionalCustomerAddressFields', 'Display additional customer address fields',
+        'This hook allows to display the extra field values added in an address from using hook ''additionalCustomerAddressFields''', '1');
```

Comment:
`''`  unwanted 🤔?

Reply:
I need this to escape single quotes in sql

Reply:
I'm getting really old ... Sorry !

Reply:
but ... if position is `tinyint(1)`, Why not 1 instead of '1' ?

Reply:
ah, I didn't modify this line (it's green because I added a coma to add my hook). All hooks seem to be added this way.